Makefile

```diff
@@ -133,15 +133,6 @@ renew: ## Refresh all relevant services that were impacted by prior Pull Request
 	$(info Refresh all relevant services that were impacted by prior Pull Request.)
 	@make db-refresh
```

Comment:
Need to refresh the DB.
Need to refresh the API to work with the new DB and endpoints.